diff -up sphinx-2.2.11-release/sphinx-min.conf.in.p1~ sphinx-2.2.11-release/sphinx-min.conf.in
--- sphinx-2.2.11-release/sphinx-min.conf.in.p1~	2024-10-22 22:59:14.968427608 +0200
+++ sphinx-2.2.11-release/sphinx-min.conf.in	2024-10-22 23:01:36.889938907 +0200
@@ -24,7 +24,7 @@ source src1
 index test1
 {
 	source			= src1
-	path			= @CONFDIR@/data/test1
+	path			= /var/lib/sphinx/test1
 }
 
 
@@ -33,7 +33,7 @@ index testrt
 	type			= rt
 	rt_mem_limit		= 128M
 
-	path			= @CONFDIR@/data/testrt
+	path			= /var/lib/sphinx/testrt
 
 	rt_field		= title
 	rt_field		= content
@@ -49,16 +49,16 @@ indexer
 
 searchd
 {
-	listen			= 9312
+	listen			= 127.0.0.1:9312
 	listen			= 9306:mysql41
-	log			= @CONFDIR@/log/searchd.log
-	query_log		= @CONFDIR@/log/query.log
+	log			= /var/log/sphinx/searchd.log
+	query_log		= /var/log/sphinx/query.log
 	read_timeout		= 5
 	max_children		= 30
-	pid_file		= @CONFDIR@/log/searchd.pid
+	pid_file		= /run/sphinx/searchd.pid
 	seamless_rotate		= 1
 	preopen_indexes		= 1
 	unlink_old		= 1
 	workers			= threads # for RT to work
-	binlog_path		= @CONFDIR@/data
+	binlog_path		= /var/log/sphinx
 }
diff -up sphinx-2.2.11-release/sphinx.conf.in.p1~ sphinx-2.2.11-release/sphinx.conf.in
--- sphinx-2.2.11-release/sphinx.conf.in.p1~	2016-07-19 12:42:18.000000000 +0200
+++ sphinx-2.2.11-release/sphinx.conf.in	2024-10-22 23:00:01.768925615 +0200
@@ -271,7 +271,7 @@ source src1
 	# shell command to invoke xmlpipe stream producer
 	# mandatory
 	#
-	# xmlpipe_command		= cat @CONFDIR@/test.xml
+	# xmlpipe_command		= cat /var/lib/sphinx/test.xml
 
 	# xmlpipe2 field declaration
 	# multi-value, optional, default is empty
@@ -338,7 +338,7 @@ index test1
 
 	# index files path and file name, without extension
 	# mandatory, path must be writable, extensions will be auto-appended
-	path			= @CONFDIR@/data/test1
+	path			= /var/lib/sphinx/test1
 
 	# document attribute values (docinfo) storage mode
 	# optional, default is 'extern'
@@ -382,13 +382,13 @@ index test1
 	# optional, default is empty
 	# contents are plain text, charset_table and stemming are both applied
 	#
-	# stopwords		= @CONFDIR@/data/stopwords.txt
+	# stopwords		= /var/lib/sphinx/stopwords.txt
 
 
 	# wordforms file, in "mapfrom > mapto" plain text format
 	# optional, default is empty
 	#
-	# wordforms		= @CONFDIR@/data/wordforms.txt
+	# wordforms		= /var/lib/sphinx/wordforms.txt
 
 
 	# tokenizing exceptions file
@@ -397,7 +397,7 @@ index test1
 	# plain text, case sensitive, space insensitive in map-from part
 	# one "Map Several Words => ToASingleOne" entry per line
 	#
-	# exceptions		= @CONFDIR@/data/exceptions.txt
+	# exceptions		= /var/lib/sphinx/exceptions.txt
 
 
 	# embedded file size limit
@@ -628,11 +628,10 @@ index test1
 # and may then be overridden in this index definition
 index test1stemmed : test1
 {
-	path			= @CONFDIR@/data/test1stemmed
+	path			= /var/lib/sphinx/test1stemmed
 	morphology		= stem_en
 }
 
-
 # distributed index example
 #
 # this is a virtual index which can NOT be directly indexed,
@@ -708,7 +707,7 @@ index rt
 
 	# index files path and file name, without extension
 	# mandatory, path must be writable, extensions will be auto-appended
-	path			= @CONFDIR@/data/rt
+	path			= /var/lib/sphinx/rt
 
 	# RAM chunk size limit
 	# RT index will keep at most this much data in RAM, then flush to disk
@@ -810,16 +809,16 @@ searchd
 	# listen			= 192.168.0.1:9312
 	# listen			= 9312
 	# listen			= /var/run/searchd.sock
-	listen			= 9312
+	listen			= 127.0.0.1:9312
 	listen			= 9306:mysql41
 
 	# log file, searchd run info is logged here
 	# optional, default is 'searchd.log'
-	log			= @CONFDIR@/log/searchd.log
+	log			= /var/log/sphinx/sphinx-searchd.log
 
 	# query log file, all search queries are logged here
 	# optional, default is empty (do not log queries)
-	query_log		= @CONFDIR@/log/query.log
+	query_log		= /var/log/sphinx/sphinx-query.log
 
 	# client read timeout, seconds
 	# optional, default is 5
@@ -840,7 +839,7 @@ searchd
 
 	# PID file, searchd process ID file name
 	# mandatory
-	pid_file		= @CONFDIR@/log/searchd.pid
+	pid_file		= /run/sphinx/searchd.pid
 
 	# seamless rotate, prevents rotate stalls if precaching huge datasets
 	# optional, default is 1
@@ -933,7 +932,7 @@ searchd
 	# optional, default is build-time configured data directory
 	#
 	# binlog_path		= # disable logging
-	# binlog_path		= @CONFDIR@/data # binlog.001 etc will be created there
+	# binlog_path		=  /var/lib/sphinx # binlog.001 etc will be created there
 
 
 	# binlog flush/sync mode
